components/teams-service/server/team.go

```diff
@@ -14,18 +14,18 @@ import (
 	"github.com/chef/automate/components/teams-service/storage"
 )
 
-// Server is a V2 teams server
-type Server struct {
+// TeamServer is a teams server
+type TeamServer struct {
```

Comment:
[nit] Does this really need exporting?

Reply:
Tried changing this and the thread ended up tugging on the whole team test setup 😅 So I think I'll turn that into a separate cleanup card

Reply:
It's fine. I thought it might be low-hanging. :shrug: